examples/with-react-intl/pages/_app.js

```diff
@@ -22,7 +22,7 @@ export default class MyApp extends App {
     // Get the `locale` and `messages` from the request object on the server.
     // In the browser, use the same values that the server serialized.
     const { req } = ctx
-    const { locale, messages } = req || window.__NEXT_DATA__.props
+    const { locale, messages } = req || window.__NEXT_DATA__.props.initialProps
```

Comment:
This seems incorrect to me

Reply:
https://github.com/zeit/next.js/blob/5d829c54f752ad9c354c017b3ddacb47a5af7a0f/examples/with-react-intl/pages/_app.js#L27 doesn't return `initialProps`, so it shouldn't be read like that afaik.